### PR TITLE
Revised README / Landing page

### PR DESCRIPTION
**Pull request by Vyrastas**
* Rewrite of README
* Split logos into individual images
* Added new image for Codelab

### CODE REVIEW

**Comment by Vyrastas** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
-# Getting started #
+OpenThread implements all features defined in the [Thread 1.1.1 specification](http://threadgroup.org/technology/ourtechnology#specifications), including all Thread networking layers (IPv6, 6LoWPAN, IEEE 802.15.4 with MAC security, Mesh Link Establishment, Mesh Routing) and device roles.
 
-The easiest way to get started is to run the CLI example in
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+Other OpenThread-specific features and enhancements include:
 
+* Network authentication with Commissioning
```
Comment (jwhui) [inline README.md]: Remove this line - this is part of the Thread Specification.

File: README.md [hunk 2]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
-`tests`       | Unit and Thread conformance tests
-`third_party` | Third-party code used by OpenThread
-`tools`       | Helpful utilities related to the OpenThread project
+* Host-side Border Agent support
+
+# Who supports OpenThread?
+
+Led by Nest, the following companies are contributing to the ongoing development of OpenThread:
+
+[![ARM][arm-logo]][arm-link][![Atmel][atmel-logo]][atmel-link][![Dialog][dialog-logo]][dialog-link][![Microsoft][ms-logo]][ms-link][![Nest][nest-logo]][nest-link][![Nordic][nordic-logo]][nordic-link][![Qualcomm][qc-logo]][qc-link][![Texas Instruments][ti-logo]][ti-link]
```
Comment (jwhui) [inline README.md]: nit: would be nice if these logos were evenly distributed in their layout like they were in the single image.

File: README.md [hunk 3]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
+[nordic-link]: http://www.nordicsemi.com/
+[qc-logo]: doc/images/ot-contrib-qc.png
+[qc-link]: https://www.qualcomm.com/
+[ti-logo]: doc/images/ot-contrib-ti.png
+[ti-link]: https://www.ti.com/
+
+### IEEE 802.15.4 Platform Support
 
+* [POSIX Emulation](https://github.com/openthread/openthread/wiki/Platforms#posix-emulation)
+* [Dialog Semiconductor DA15000 OpenThread Sandbox Development Kit](https://github.com/openthread/openthread/wiki/Platforms#dialog-da15000)
```
Comment (jwhui) [inline README.md]: Might be better just to list the SoCs.  For example:

- Dialog Semiconductor DA15000
- Nordic Semiconductor nRF52840
- Texas Instruments CC2538
- Texas Instruments CC2650

File: README.md [hunk 4]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
+### Testing
 
- *  [openthread-announce](https://groups.google.com/forum/#!forum/openthread-announce)
-    \- subscribe for release notes and new updates on OpenThread
- *  [openthread-users](https://groups.google.com/forum/#!forum/openthread-users)
-    \- the best place for users to discuss OpenThread and interact with
-    the OpenThread team
- *  [openthread-devel](https://groups.google.com/forum/#!forum/openthread-devel)
-    \- team members discuss the on-going development of OpenThread
+Testing is done with the [GRL Thread Test Harness software](http://graniteriverlabs.com/thread/), available for download to Thread member companies.
```
Comment (jwhui) [inline README.md]: nit: Certification testing is done...

File: README.md [hunk 5]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
+* **Windows 10** — [universal drivers](https://github.com/openthread/openthread/wiki/OpenThread-on%C2%A0Windows) to interface with devices running OpenThread
 
-# Getting help #
+# What tools are available for testing?
 
-Submit bugs and feature requests to [issue tracker][ot-issues]. Usage
-questions? Post your questions to [Stack Overflow][stackoverflow] using the
-[`openthread` tag][ot-tag]. We also use Google Groups for discussion
-and announcements:
+### Testing
```
Comment (jwhui) [inline README.md]: Change to "Certification Testing"?

File: README.md [hunk 6]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
 
-The Doxygen reference docs are [hosted online][ot-docs] and generated
-as part of the build.
+### Desktop Support
 
-[ot-docs]: http://openthread.github.io/openthread/
+Desktop platforms can also be used to control and interface with a Thread network using OpenThread:
 
+* **Unix** — [`wpantund`](https://github.com/openthread/wpantund) provides an interface to an NCP
+* **Windows 10** — [universal drivers](https://github.com/openthread/openthread/wiki/OpenThread-on%C2%A0Windows) to interface with devices running OpenThread
```
Comment (jwhui) [inline README.md]: @nibanks, is this the right terminology?

File: README.md [hunk 7]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
-for more details.
+Other OpenThread-specific features and enhancements include:
 
+* Network authentication with Commissioning
+* IPv6 configuration and raw data interface
+* UDP sockets
+* CoAP client and server
+* DHCPv6 client and server
+* DNSv6 client
+* Command Line Interface (CLI)
```
Comment (jwhui) [inline README.md]: Can we group the above items under: "Application Support and Services"?

File: README.md [hunk 8]
```diff
@@ -29,104 +30,158 @@ applications. More information about Thread can be found on
...
+* Network Co-Processor (NCP) support - off-loading Thread functions to a SoC
+* Spinel - general purpose NCP protocol
+* `wpantund` - user-space NCP network interface driver/daemon
+* Sniffer support via NCP Spinel nodes
```
Comment (jwhui) [inline README.md]: Can we group the above items under: "Network Co-Processor Support"?  Maybe include a line somewhere like "OpenThread supports both single-on-chip (SoC) and network co-processor (NCP) designs."

File: README.md [hunk 9]
```diff
@@ -5,19 +5,20 @@
...
 <img src="https://cdn.rawgit.com/openthread/openthread/ab4c4e1e/doc/images/certified.svg" alt="Thread Certified Component" width="150px" align="right">
 </a>
 
-OpenThread is an open-source implementation of the [Thread][thread]
-networking protocol. Nest has released OpenThread to make the technology
-used in Nest products more broadly available to developers to accelerate
-the development of products for the connected home.
+**...an open-source implementation of the [Thread](http://threadgroup.org/technology/ourtechnology) networking protocol.** Nest has released OpenThread to make the technology used in Nest products more broadly available to developers to accelerate the development of products for the connected home.
+
+**...OS and platform agnostic**, with a radio abstraction layer and a small memory footprint, making it highly portable.
```
Comment (jwhui) [inline README.md]: radio abstraction -> narrow platform abstraction

File: README.md (hunk 2 above)
Comment (beriberikix) [inline README.md]: Github doesn't provide the best tools for layout and formatting. You can hack it like I did for the Thread Certified Component image, though.

Comment (jwhui) [inline README.md]: How about making the images themselves equal size?

Comment (Vyrastas) [inline README.md]: We can go either way.  I'll pick one and make sure it's 4 to a row like before.

File: README.md (hunk 3 above)
Comment (Vyrastas) [inline README.md]: @jwhui Where should I mention POSIX Emulation then... as its own independent thing or under Desktop Support?

Comment (jwhui) [inline README.md]: I think we can keep it in this list.

Comment (Vyrastas) [inline README.md]: I'll just move it to the end of the list, then.

**Comment by Vyrastas** [comment]
Addressed everything but the Windows 10 terminology question.

File: README.md [hunk 10]
```diff
@@ -29,104 +30,143 @@ applications. More information about Thread can be found on
...
+
+OpenThread has been ported to several devices and platforms. Build examples are included in the OpenThread library.
+
+### IEEE 802.15.4 Platform Support
 
+* [Dialog Semiconductor DA15000 OpenThread Sandbox Development Kit](https://github.com/openthread/openthread/wiki/Platforms#dialog-da15000)
+* [Nordic Semiconductor nRF52840 SoC](https://github.com/openthread/openthread/wiki/Platforms#nordic-semiconductor-nrf52840)
+* [Texas Instruments SimpleLink CC2538](https://github.com/openthread/openthread/wiki/Platforms#texas-instruments-cc2538)
+* [Texas Instruments SimpleLink CC2650](https://github.com/openthread/openthread/wiki/Platforms#texas-instruments-cc2650)
```
Comment (jwhui) [inline README.md]: I think we can simplify the above to just include the chip name:

- Dialog Semiconductor DA15000
- Nordic Semiconductor nRF52840
- Texas Instruments CC2538
- Texas Instruments CC2650

File: README.md [hunk 11]
```diff
@@ -29,104 +30,143 @@ applications. More information about Thread can be found on
...
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+OpenThread also supports both single-on-chip (SoC) and network co-processor (NCP) designs.
 
+Other features and enhancements include:
 
-## What's included ##
+* IPv6 configuration and raw data interface
+* UDP sockets
```
Comment (jwhui) [inline README.md]: I would include the IPv6 and UDP line items in the "Application support" section below.

File: README.md [hunk 12]
```diff
@@ -29,104 +30,141 @@ applications. More information about Thread can be found on
...
+# OpenThread Features
 
-# Getting started #
+OpenThread implements all features defined in the [Thread 1.1.1 specification](http://threadgroup.org/technology/ourtechnology#specifications), including all Thread networking layers (IPv6, 6LoWPAN, IEEE 802.15.4 with MAC security, Mesh Link Establishment, Mesh Routing) and device roles.
 
-The easiest way to get started is to run the CLI example in
-`/examples/apps/cli`. See the
-[CLI example README](examples/apps/cli/README.md)
-for more details.
+OpenThread supports both single-on-chip (SoC) and network co-processor (NCP) designs. Other features and enhancements include:
```
Comment (jwhui) [inline README.md]: single-on-chip -> system-on-chip

File: README.md [hunk 13]
```diff
@@ -29,104 +30,141 @@ applications. More information about Thread can be found on
...
-    \- the best place for users to discuss OpenThread and interact with
-    the OpenThread team
- *  [openthread-devel](https://groups.google.com/forum/#!forum/openthread-devel)
-    \- team members discuss the on-going development of OpenThread
+Certification testing is done with the [GRL Thread Test Harness software](http://graniteriverlabs.com/thread/), available for download to Thread member companies.
 
-[ot-issues]: https://github.com/openthread/openthread/issues
-[stackoverflow]: http://stackoverflow.com/
-[ot-tag]: http://stackoverflow.com/questions/tagged/openthread
+Additional tools that extend the Test Harness are included in the OpenThread library:
```
Comment (jwhui) [inline README.md]: library -> project

**Comment by beriberikix** [comment]
w00t! Great update, @Vyrastas!

**Pull request merged**